src/app-root.ts

```diff
@@ -489,6 +538,50 @@ export class AppRoot extends LitElement {
     }
   }
 
+  private async showEmptyPlaceholder(placeholderType: string) {
+    switch (placeholderType) {
+      case 'empty query':
```

Comment:
I know we are on demo for developers but we should not use string slugs for checking conditions. I would write these something like `empty-query`, `empty-collection`, `empty-profile` something like this.